src/main/java/spoon/support/visitor/equals/EqualsChecker.java

```diff
@@ -254,4 +255,33 @@ public void visitCtContinue(CtContinue e) {
 		}
 		super.visitCtLiteral(e);
 	}
+
+	@Override
+	public void visitCtComment(CtComment e) {
```

Comment:
I don't see any equals method in [CtComment](https://github.com/INRIA/spoon/blob/master/src/main/java/spoon/reflect/code/CtComment.java)